client/src/details/opportunitydetails.cpp

```diff
@@ -48,9 +48,7 @@ OpportunityDetails::OpportunityDetails(QWidget *parent)
     mUi->setupUi(this);
 
     mUi->urllabel->setTextInteractionFlags(Qt::LinksAccessibleByMouse);
-    mUi->next_call_date->calendarWidget()->setVerticalHeaderFormat(QCalendarWidget::ISOWeekNumbers);
```

Comment:
(does the new widget show week numbers too?)

Reply:
yes it shows in a combo box, and allow you to pick a week